gamemodes/terrortown/entities/entities/ttt_flame.lua

```diff
@@ -13,54 +13,74 @@ ENT.Model = Model("models/weapons/w_eq_flashbang_thrown.mdl")
 -- @realm shared
 AccessorFunc(ENT, "dmgparent", "DamageParent")
 
----
--- @accessor boolean
--- @realm shared
-AccessorFunc(ENT, "die_explode", "ExplodeOnDeath")
+ENT.firechild = nil
 
----
--- @accessor number
--- @realm shared
-AccessorFunc(ENT, "dietime", "DieTime")
+ENT.fire_ignite_delay = 0
 
-ENT.firechild = nil
-ENT.fireparams = {
-	size = 120,
-	growth = 1
-}
+ENT.real_scale = nil
+ENT.fire_scale_base = 0.9
+ENT.fire_scale_variance = 0.2
 
 ENT.next_hurt = 0
 ENT.hurt_interval = 1
 
+ENT.low_detail_fps = 12
+ENT.low_detail_size = 10
+
+ENT.hurt_radius = nil
+ENT.hurt_base = 5
+ENT.hurt_variance = 1
+ENT.hurt_radius_scale_factor = 0.4
+
+ENT.trail = nil
+ENT.trail_color = Color(255, 56, 56, 255)
+ENT.trail_width = 32
+ENT.trail_end_width = 0
+ENT.trail_lifetime = 0.25
+ENT.trail_res = 16
+ENT.trail_texture = "trails/physbeam"
+
+ENT.debug_viz = false
+
 ---
 -- @realm shared
 function ENT:SetupDataTables()
 	self:NetworkVar("Bool", 0, "Burning")
+	self:NetworkVar("Bool", 1, "Immobile")
+	self:NetworkVar("Bool", 2, "ExplodeOnDeath")
+	self:NetworkVar("Float", 0, "FlameSize")
+	self:NetworkVar("Float", 1, "LifeSpan")
+	self:NetworkVar("Float", 2, "DieTime")
 end
 
 ---
 -- @realm shared
 function ENT:Initialize()
 	self:SetModel(self.Model)
 	self:DrawShadow(false)
-	self:SetNoDraw(true)
-
-	if CLIENT and GetConVar("ttt_fire_fallback"):GetBool() then
-		self.Draw = self.BackupDraw
-		self:SetNoDraw(false)
-	end
+	self:SetNoDraw(false)
 
 	self:PhysicsInit(SOLID_VPHYSICS)
 	self:SetMoveType(MOVETYPE_VPHYSICS)
 	self:SetSolid(SOLID_VPHYSICS)
 	self:SetCollisionGroup(COLLISION_GROUP_DEBRIS)
 	self:SetHealth(99999)
 
+	if SERVER then
+		self.trail = util.SpriteTrail( self, 0, self.trail_color, true, self.trail_width, self.trail_end_width, self.trail_lifetime, self.trail_lifetime, self.trail_texture )
+		self:DeleteOnRemove(self.trail)
+	end
```

Comment:
it was added because these flame entities are invisible until they find a resting position, fling far away from their point of origin, and those issues compounded the flames not matching the size of their hurtbox

they look like a short red trail

Reply:
I am kind of against it from my imagination. I'd need a video but otherwise i tend to suggest removing this, dunno what @TimGoll or @Histalek thinks about this.

Reply:
You could pull the branch and see it in motion, isn't that why we make these small? So people can look at them? I'm sure I posted lots of footage of this over the last 6 months.

https://github.com/TTT-2/TTT2/assets/5711436/d52748f3-b7b9-469a-b61f-5f93a0f91cb2

Reply:
I could but i am reviewing many PRs today and adding such videos to the PR descriptions helps a lot to speed up discussion about such changes.

But yes this seems fine to me thanks!